app/models/session.rb

```diff
@@ -4,6 +4,10 @@ def expired?
     || (updated_at.utc + ttl.seconds) < Time.zone.now.utc
   end
 
+  # def expired?
+  #   expires_at < Time.zone.now.utc
+  # end
+
```

Comment:
can we drop this method?

Reply:
its going to be uncommented in a later commit. i was using it as a placeholder to show what the api will be